JavaScriptTasks/Session 5 - OOPInheritance(5-3-2020)/ArrayHelpers.js

```diff
@@ -0,0 +1,62 @@
+// PROBLEM STATEMENT :Array Helpers
+
+// http://www.codewars.com/kata/array-helpers
+// SOLUTION:
+
+  // TODO
+//var numbers = [1, 2, 3, 4, 5];
+Array.prototype.square = function(){
```

Comment:
before adding any method to prototype, please check whether that method is already exists on prototype or not

Reply:
updated code . Added check using hasOwnProperty on prototype before adding the method.